tlsn/examples/discord/README.md

```diff
@@ -104,3 +104,6 @@ cargo run --release --example discord_dm_verifier
 ```
 
 This will verify the proof and print out the redacted transcript!
+
```

Comment:
Need to modify L10 in this doc as it still refers to the now deleted `simple_notary.rs`

> P/S: The notary server used in this example is more functional compared to its [simple version](../simple/simple_notary.rs).